ouroboros-consensus/ouroboros-consensus-mock/test/Test/ThreadNet/LeaderSchedule.hs

```diff
@@ -99,32 +95,44 @@ prop_simple_leader_schedule_convergence
       , pgaFirstBlockNo       = 0
       , pgaFixedMaxForkLength = Nothing
       , pgaFixedSchedule      = Just schedule
-      , pgaSecurityParam      = praosSecurityParam
+      , pgaSecurityParam      = k
       , pgaTestConfig         = testConfig
+      , pgaTestConfigB        = testConfigB
       }
       testOutput
   where
+    TestConfig{numCoreNodes} = testConfig
+
+    testConfigB = TestConfigB
+      { epochSize
+      , forgeEbbEnv  = Nothing
+      , nodeJoinPlan
+      , slotLength
+      , nodeRestarts = noRestarts
+      , txGenExtra   = ()
+      }
+
+    -- this is entirely ignored because of the 'WithLeaderSchedule' combinator
+    dummyF = 0.5
+
     testOutput@TestOutput{testOutputNodes} =
-        runTestNetwork testConfig epochSize TestConfigBlock
-            { forgeEbbEnv = Nothing
-            , nodeInfo    = \nid -> plainTestNodeInitialization $
-                                    protocolInfoPraosRule
-                                      numCoreNodes
-                                      nid
-                                      params
-                                      (HardFork.defaultEraParams
-                                        praosSecurityParam
-                                        praosSlotLength)
-                                      schedule
-            , rekeying    = Nothing
-            , txGenExtra  = ()
+        runTestNetwork testConfig testConfigB TestConfigMB
+            { nodeInfo = \nid ->
+                plainTestNodeInitialization $
+                protocolInfoPraosRule
+                  numCoreNodes
+                  nid
+                  PraosParams
+                  { praosSecurityParam = k
+                  , praosSlotsPerEpoch = unEpochSize epochSize
+                  , praosLeaderF       = dummyF
+                  , praosLifetimeKES   = 1000000
+                  }
+                  (HardFork.defaultEraParams k slotLength)
+                  schedule
+            , mkRekeyM = Nothing
             }
 
-    -- I don't think this value actually matters if we override the leader
```

Comment:
Comment intentionally deleted? It still applies, right?

Reply:
I've put it back now. I think I replaced it because it was previously "justifying" using a fixed value, but now I've included it in the generator -- so we're sweeping its values. But, it is still good to comment saying that we expect its value should not matter.